featuretools/tests/entityset_tests/test_serialization.py

```diff
@@ -59,12 +59,22 @@ def test_entityset_description(es):
     assert es.metadata.__eq__(_es, deep=True)
 
 
-def test_invalid_formats(es):
+@pytest.fixture
+def path_management():
     path = os.path.join(CACHE, 'es')
+    try:
+        os.makedirs(path)
+    except (OSError, IOError):
```

Comment:
We want to ignore the error raised by `os.makedirs` if the folder already exists but let any unexpected errors get raised.  `os.makedirs` raises an OSError in both python 2 and 3 so we should only catch OSError

Reply:
I changed it to catch `OSError` and then check the error code of that error to be `errno.EEXIST` but I can't figure out how to order the imports correctly for `import errno`. I checked the pep8 guidelines and it's with the standard library imports but it still fails the make lint. I tried it in a bunch of different specific places as well.